ui/components/ui/dropdown/dropdown.scss

```diff
@@ -3,7 +3,7 @@
 .dropdown {
   position: relative;
   display: inline-block;
-  height: 36px;
+  height: 48px;
```

Comment:
We made this change on component level
cc @amandaye0h

Reply:
It may be worth doing a check of all the instances that this might effect to ensure there are no visual regressions